### ISSUE TITLE
Modal windows do not have keyboard focus

### ISSUE THREAD

**Issue by bokicoder** [AvengeMedia/DankMaterialShell#377]
## Compositor

- [x] niri
- [ ] Hyprland
- [ ] Other (specify)

## Description

Multiple modal windows fail to receive keyboard input. The most severe issue I've found is with the clipboard, and notifications also have some problems. There might be more windows affected.

Related https://github.com/AvengeMedia/DankMaterialShell/issues/375

**Comment by bokicoder** [comment]
Focusing can be achieved by clicking various buttons on the window, but it works sometimes and not others

**Comment by bbedward** [comment]
I think taken care of 01a42ff33008507eabb6a04537a2ce21f51b3b59

**Comment by bokicoder** [comment]
There is an issue with the notification window; it doesn't have keyboard focus the first time it's opened, but closing and reopening it resolves the problem

**Comment by bokicoder** [comment]
The clipboard is working perfectly fine

**Comment by bokicoder** [comment]
Additionally: The notification window refers to the window displayed by the `dms ipc call notifications toggle` command

**Comment by bbedward** [comment]
Think - this is fixed 080b7a28b125c04ef32811a50188e8ed926642d1

**Comment by bokicoder** [comment]
Okay, I've found another issue with the clipboard. After I click the clear icon in the upper-right corner of the clipboard window and then confirm in the confirmation dialog, the clipboard window loses keyboard focus again.

**Comment by bokicoder** [comment]
> Okay, I've found another issue with the clipboard. After I click the clear icon in the upper-right corner of the clipboard window and then confirm in the confirmation dialog, the clipboard window loses keyboard focus again.

@bbedward Would you mind fixing this?
